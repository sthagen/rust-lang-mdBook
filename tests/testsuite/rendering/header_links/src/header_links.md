# Header Links

## Foo^bar

###

####

## HÃ¯

## Repeat
## Repeat
## Repeat

## Repeat 1

## <!--comment--> With *emphasis* **bold** **_bold_emphasis_** `code` \<escaped> <span>html</span> [link] <https://example.com/>

[link]: https://example.com/link
