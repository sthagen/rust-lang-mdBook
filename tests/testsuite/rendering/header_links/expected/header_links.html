<h1 id="header-links"><a class="header" href="#header-links">Header Links</a></h1>
<h2 id="foobar"><a class="header" href="#foobar">Foo^bar</a></h2>
<h3 id=""><a class="header" href="#"></a></h3>
<h4 id="-1"><a class="header" href="#-1"></a></h4>
<h2 id="hï"><a class="header" href="#hï">Hï</a></h2>
<h2 id="repeat"><a class="header" href="#repeat">Repeat</a></h2>
<h2 id="repeat-1"><a class="header" href="#repeat-1">Repeat</a></h2>
<h2 id="repeat-2"><a class="header" href="#repeat-2">Repeat</a></h2>
<h2 id="repeat-1"><a class="header" href="#repeat-1">Repeat 1</a></h2>
<h2 id="with-emphasis-bold-bold_emphasis-code-escaped-html-link-httpsexamplecom"><a class="header" href="#with-emphasis-bold-bold_emphasis-code-escaped-html-link-httpsexamplecom"><!--comment--> With <em>emphasis</em> <strong>bold</strong> <strong><em>bold_emphasis</em></strong> <code>code</code> &lt;escaped&gt; <span>html</span> <a href="https://example.com/link">link</a> <a href="https://example.com/">https://example.com/</a></a></h2>